flux.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: flux
+  version: 0.38.2
+  epoch: 0
+  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - build-base
+      - bash
+      - curl
+      - go
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/fluxcd/flux2/archive/v${{package.version}}/v${{package.version}}.tar.gz
+      expected-sha256: 6f64ac68748137816e85b2258681782951b04135d00d8e69e0efa96438f47177
+  - runs: |
+      curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
We should probably package customize instead of curling it in here.

Reply:
kustomize has been added to wolfi now https://github.com/wolfi-dev/os/blob/main/kustomize.yaml so yeah we should try adding it as an `environment.contents.package`

Reply:
yep, I've added kustomize to the package list.